opm/material/components/Brine.hpp

```diff
@@ -329,23 +328,39 @@ class Brine : public Component<Scalar, Brine<Scalar, H2O> >
 
     /*!
      * \copydoc H2O::liquidViscosity
-     *
-     * Equation given in:
-     * - Batzle & Wang (1992)
-     * - cited by: Bachu & Adams (2002)
-     *   "Equations of State for basin geofluids"
+     * Modified Jones - Doles equation to account for temperature effects
+     * Coeficients from McBride-Wright, 2013
+     * Viscosity and density of aqueous fluids with dissolved CO2"
      */
     template <class Evaluation>
-    static Evaluation liquidViscosity(const Evaluation& temperature, const Evaluation& /*pressure*/)
+    static Evaluation liquidViscosity(const Evaluation& temperature, const Evaluation& pressure)
     {
-        Evaluation T_C = temperature - 273.15;
-        if(temperature <= 275.) // regularization
-            T_C = 275.0;
-
-        Evaluation A = (0.42*std::pow((std::pow(salinity, 0.8)-0.17), 2) + 0.045)*pow(T_C, 0.8);
-        Evaluation mu_brine = 0.1 + 0.333*salinity + (1.65+91.9*salinity*salinity*salinity)*exp(-A);
-
-        return mu_brine/1000.0; // convert to [Pa s] (todo: check if correct cP->Pa s is times 10...)
+        // Modified Jones - Doles equation to account for temperature effects
+        // Coefficients from McBride-Wright, 2013
+        // "Viscosity and density of aqueous fluids with dissolved CO2"
+        std::vector<double> c = {0.291015724, -0.119318366, 0.190911976};
+        const Scalar MmNaCl = 58e-3; // molar mass of NaCl [kg/mol]
```

Comment:
The molar mass of NaCl has different precision here from what was used in the previous PR related to CO2STORE. I think it would be nice to be consistent in the code.